scripts/microbe_stage/setup.as

```diff
@@ -255,6 +255,12 @@ void onReturnFromEditor(CellStageWorld@ world)
     MicrobeComponent@ microbeComponent = cast<MicrobeComponent>(
         world.GetScriptComponentHolder("MicrobeComponent").Find(player));
 
+    // Update physics of player cell
+    auto rigidBody = world.GetComponent_Physics(player);
+    auto shape = world.GetPhysicalWorld().CreateCompound();
+
+    MicrobeOperations::_applyMicrobeCollisionShape(world, rigidBody, microbeComponent, shape);
```

Comment:
I don't understand why this is working. You are basically creating an empty shape and setting that as the player's collision. Why does this work? I want to resolve the issue with `Species::restoreOrganelleLayout` not doing the right thing here.

Reply:
Now that I thought about this a bit more, could it be the case that as when the cell is a bacteria the physics body is a sphere (and not compound body) so then when the physics collision tries to update it runs into a script error (did you check if you remove this change do you get script errors?). If that's the case then the organelle layout restore should be updated to automatically recreate a compound shape if it is "upgrading" a bacteria.